packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -0,0 +1,109 @@
+import designSystemDefaults, { IDesignSystem } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { applyLocalizedProperty, Direction, toPx } from "@microsoft/fast-jss-utilities";
+import { IMSFTProgressClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import Chroma from "chroma-js";
+import { get } from "lodash";
+
+/* tslint:disable-next-line */
+const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IMSFTProgressClassNameContract, IDesignSystem> => {
+    const backgroundColor: string = get(config, "backgroundColor") || designSystemDefaults.backgroundColor;

```

Comment:
Can you give using the contrast helpers a shot? Ping me if you have questions on them.